keyboards/planck/keymaps/dr_notsokind/keymap.c

```diff
@@ -1,6 +1,5 @@
 // Layout picture at http://www.keyboard-layout-editor.com/#/gists/125febfad6960add078e6f14256539b6
 
-#pragma message "You may need to add LAYOUT_planck_grid to your keymap layers - see default for an example"
 #include "planck.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```